### DISCUSSION TITLE
video sites can't download video

### DISCUSSION THREAD

**Discussion by HungJerry** [aclap-dev/video-downloadhelper#245] [Help and Q&A]
### Your question

https://www.ptsplus.tv/zh/programs/d6b1cf47-62e1-454a-977b-ab5021b698e0?play=e845ea11-a86e-44dc-940f-90ff8a2f8820
login Facebook can play video, but  can not download video

Exit code: 1

moz-extension://8f20a7ca-cbf5-4a8e-bf27-acc8b47879ba/background/main.js:1
receive@moz-extension://8f20a7ca-cbf5-4a8e-bf27-acc8b47879ba/background/main.js:1:13157
Zp</callCatchAppNotFound/</<@moz-extension://8f20a7ca-cbf5-4a8e-bf27-acc8b47879ba/background/main.js:1:42626

### Addon & CoApp version and details

Video DownloadHelper
版本 8.2.1.1 (stable)
瀏覽器地區: zh-TW
穩定版
編譯時間: 2024-02-19
編譯選項: browser=firefox, linuxlic=false, noyt=false, target=mozilla
平台 Win x86-64
瀏覽器 Mozilla Firefox 123.0
夥伴軟體版本: VdhCoApp 2.0.19
夥伴軟體執行檔: C:\Program Files\DownloadHelper CoApp\vdhcoapp.exe

**Comment by erwin54321** [comment]
I have the same problem since 27.02.24 (first line is the video - the the error)

https://www.youtube.com/watch?v=Mhp3spg4TrE

Exit code: 1

moz-extension://d6a6a2a0-eb87-44c3-9b65-ea1b2498eeb8/background/main.js:1
receive@moz-extension://d6a6a2a0-eb87-44c3-9b65-ea1b2498eeb8/background/main.js:1:13157
Zp</callCatchAppNotFound/</<@moz-extension://d6a6a2a0-eb87-44c3-9b65-ea1b2498eeb8/background/main.js:1:42626

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: false, browser: firefox
lang: de
license: accepted
key (only 16 first characters): cd391840570c48a5
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Uwe Timm",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by GrampaWildWilly** [comment]
I am running Windows 11 Pro 64-bit version 23H2 build 22631.3235, Firefox 123.0 64-bit, licensed VDH 8.2.2.0a1 beta, CoApp 2.0.19.  I do not work for VDH.  I'm just another user.

The ptsplus site requires a login.  I can't investigate that one.

The YouTube video downloaded no problem for me.  I downloaded both the 1920x1080 mp4 & the 1920x1080 mkv.  They each took a bit over a minute to download.  The mp4 was slightly smaller than the mkv.  Both files played fine for the full 1:00:11 of their duration, video & audio all the way to the end.  I didn't sit & watch them, just sampled them at intervals.  Everything looked fine.

Conclusion: Get the latest beta of VDH.  Click the Wiki link at the top of this page.  Then look in the Table of Contents on the right side of the Wiki page for the chapter on beta versions.  You'll find the link there to the VDH beta.

**Comment by Mattmjs** [comment]
@HungJerry 
Can you share the master manifest and the media manifest found in the details page.

**Reply by HungJerry** [reply]
[1080p.pdf](https://github.com/aclap-dev/video-downloadhelper/files/14546739/1080p.pdf)
[720p.pdf](https://github.com/aclap-dev/video-downloadhelper/files/14546740/720p.pdf)
[480p.pdf](https://github.com/aclap-dev/video-downloadhelper/files/14546741/480p.pdf)

**Comment by HungJerry** [comment]
[1080p.pdf](https://github.com/aclap-dev/video-downloadhelper/files/14546735/1080p.pdf)
[720p.pdf](https://github.com/aclap-dev/video-downloadhelper/files/14546736/720p.pdf)
[480p.pdf](https://github.com/aclap-dev/video-downloadhelper/files/14546737/480p.pdf)


> @HungJerry Can you share the master manifest and the media manifest found in the details page.

**Reply by Mattmjs** [reply]
It's hard to say what the issue is with that information, maybe there is some protection on your site.
You should also verify anti virus & firewall are not blocking the companion app.

**Reply by erwin54321** [reply]
You are wright! I testet div. changes an anti virus Bitdefender toatl security and found, it blocked "dangerous" sides. They seemd to have changes in ther blocking, so from one day to another vied-downlodhleper doesn't work on sides, that before were working fine. In Bitdefendr there is virus-protection (no problem), but there is "expanded danger prevention" and "online danger prevention" what makes the problemes. Thank You for Your tip.
